bot/cogs/snekbox.py

```diff
@@ -43,6 +46,25 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.jobs = {}
 
+        # Dict[int[user_id],
+        #      Tuple[str[saved_code],
+        #            datetime[last_used]]]
```

Comment:
Don't randomly leave commented out code in your PRs.

Reply:
This isn't commented out code, this is a comment explaining the type used so other people working on this know what it is.